addon/templates/components/frost-bunsen-section.hbs

```diff
@@ -18,7 +18,7 @@
       {{title}}
     </h3>
   {{/if}}
-  {{#if required}}
+  {{#if showRequiredLabel}}
```

Comment:
I don't think `frost-bunsen-section` has the `showRequiredLabel` computed property does it?

Reply:
No - you are right.